kedro/framework/project/__init__.py

```diff
@@ -54,8 +54,8 @@ def validator_func(settings, validators):
 class _IsSubclassValidator(Validator):
     """A validator to check if the supplied setting value is a subclass of the default class"""
 
-    def _validate_items(self, settings, env=None):
-        super()._validate_items(settings, env)
+    def validate(self, settings, *args, **kwargs):
+        super().validate(settings, *args, **kwargs)
```

Comment:
Do we still need to pass `env=None` ourselves or is it a default for `validate()` too?

Reply:
It's passed internally by dynaconf. It's set as an attribute on the `self` of the `Validator`.